qubes-kde-dom0/qubes-kde-dom0.spec

```diff
@@ -23,8 +23,9 @@ Requires: kde-filesystem
 Requires: kde-settings
 Requires: kde-settings-kdm
 Requires: kdelibs >= %{version}
-Requires: kde-workspace
-Requires: kde-workspace-libs
+Requires: kde-workspace-common
+#Requires: kde-workspace
+#Requires: kde-workspace-libs
```

Comment:
If those packages are no longer in Fedora 23, better remove them completely. But first check if wasn't simply renamed. `dnf groupinfo kde-desktop` may be helpful, for example I see there `plasma-workspace`.